.github/workflows/ci-docker.yaml

```diff
@@ -19,12 +19,21 @@ jobs:
         uses: docker/setup-qemu-action@v2
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v2
-      - name: Build
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build and Push
         uses: docker/build-push-action@v3
         with:
-          context: .
+          context: .github/docker
```

Comment:
This doesn't seem quite right.  I think we want the build to run in the root of the repo